i915.c

```diff
@@ -492,8 +492,8 @@ static int i915_add_combinations(struct driver *drv)
 static int i915_align_dimensions(struct bo *bo, uint32_t format, uint32_t tiling, uint32_t *stride,
 				 uint32_t *aligned_height)
 {
-	uint32_t horizontal_alignment = 64;
-	uint32_t vertical_alignment = 4;
+	uint32_t horizontal_alignment;
```

Comment:
Are you sure you want to leave these values uninitialized?

Reply:
yes. the value of horizontal_alignment gets updated on every condition mentioned here. And whatever we initialize here, will never be used. That was the reason it caused coverity error "Unused Value".